### DISCUSSION TITLE
How do I import an sqlite map?

### DISCUSSION THREAD

**Discussion by geoffmozz** [osmandapp/OsmAnd#11231] [Q&A]
I have a pre-built sqlite map I'd like to try in OsmAnd.

How do I add the map to the app?

I'm on Android 11   OsmAnd 3.9

Thanks...

**Comment by vshcherb** [comment]
It should be part of documentation - https://docs.osmand.net/en/main@latest/osmand/map/online-raster-maps#prepare--copy-raster-maps-to-device

**Reply by geoffmozz** [reply]
Thanks for your help. The Android folder specified is inaccessible (I'm on Android 11) I can see the folder and files when the phone is plugged into the PC but the Android folder is 'locked'. I can't drag and drop files to it :-(  Any idea how to get round this?

**Comment by tommycrock** [comment]
Probably change the storage to the shared space. I'm guessing your osmand
folder is in Android/data? If so, change in Settings, Osmand settings, data
storage folder.  Android 11 has locked those folders away.

**Reply by geoffmozz** [reply]
Good idea....thanks :-)

**Reply by mariush444** [reply]
Just use osf. It can store even in private storage.

**Comment by vascotenner** [comment]
I am running in the same issue. On https://www.osmand.net/docs/user/map/raster-maps/#prepare--copy-raster-maps-to-device is mentioned:
"Menu → Download maps → Local → Online and cached tile maps → choose online maps →
⋮ → Edit → Storage format → SQLiteDB file / One image file per tile"

I do not see a local button in downloaded maps.

On my old device I could directly open .sqlitedb files that I had downloaded from my "android" file explorer, it gave the option to open in osmand.

I found this solution: Change data directory:
menu -> settings -> settings for osmand -> Data directory ->  Directory for multiple yusers (`/storatge/emulated/0/Android/obb.net.osmand.plus/tiles`

And move .sqlitedb files to:
```/storatge/emulated/0/Android/obb.net.osmand.plus/tiles```
